rest-api-spec/build.gradle

```diff
@@ -104,7 +104,6 @@ tasks.named("yamlRestCompatTest").configure {
     'mtermvectors/11_basic_with_types/Basic tests for multi termvector get',
     'mtermvectors/21_deprecated_with_types/Deprecated camel case and _ parameters should fail in Term Vectors query',
     'mtermvectors/30_mix_typeless_typeful/mtermvectors without types on an index that has types',
-    'search/10_source_filtering/docvalue_fields with default format', //use_field_mapping change
```

Comment:
the 7.x test asserts about the deprecation warning and the right format (declared on mapping) being used